dkarlovi/symfony-flex-docker/0.1/Makefile

```diff
@@ -0,0 +1,35 @@
+docker-start:
+	docker-compose -f ./docker-compose.yaml -f ./.infra/docker-compose/docker-compose.$(APP_ENV).yaml up -d
+.PHONY: docker-up
+
+docker-stop:
```

Comment:
I know, I've named targets `start` & `stop` because that's what I expect non-`docker-compose` users to look for first, but the difference between `up`/`start`, `down`/`stop` is not significant enough here.

Also, from my personal experience, I've never once used `start`/`stop`, only `up`/`down.